### ISSUE TITLE
Color Contrast failure and hidden text issue identified in template

### ISSUE THREAD

**Issue by vprentice** [Section508Coordinators/DHSA-TT-200-A#10]
At the bottom of the template page there is hidden text that fails to meet the required 4.5.1 ratio, it is shown to have a 3.1 color contrast ratio. When testing with the ANDi structure tool, then using the Headings option, the following text is identified <h4>DHS Accessibility Help Desk</h4> at the bottom of the page that is visually hidden.

**Comment by katbecker** [comment]
I've changed that footing heading color so it shows up now (shouldn't have been hidden).
